soc/arm/nxp_imx/mcimx7_m4/Kconfig.series

```diff
@@ -6,6 +6,7 @@
 config SOC_SERIES_IMX7_M4
 	bool "i.MX7 M4 Core Series"
 	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
```

Comment:
Add this to mcimx6x_m4 as well.
Reference: IMX6SXRM Chapter 13.7 ARM Cortex M4 Platform/Debug Configuration

Reply:
Done @agansari , thanks for taking a look